mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -17,7 +17,7 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	FIELD field_1700 instance Lnet/minecraft/class_310;
 	FIELD field_1701 openProfilerSection Ljava/lang/String;
 	FIELD field_1702 toastManager Lnet/minecraft/class_374;
-	FIELD field_1703 IS_SYSTEM_MAC Z
+	FIELD field_1703 SYSTEM_MAC Z
```

Comment:
Sounds kinda odd. Is there a better way to word this?

Reply:
CURRENT_SYSTEM_MAC?

Reply:
yea that sounds better